### PR TITLE
Bump kind dependency

### PR DESCRIPTION
**Pull request by awgreene**
**Description of the change:**

**Motivation for the change:**

**Architectural changes:**

**Testing remarks:**

**Reviewer Checklist**
- [ ] Implementation matches the proposed design, or proposal is updated to match implementation
- [ ] Sufficient unit test coverage
- [ ] Sufficient end-to-end test coverage
- [ ] Bug fixes are accompanied by regression test(s)
- [ ] e2e tests and flake fixes are accompanied evidence of flake testing, e.g. executing the test 100(0) times
- [ ] tech debt/todo is accompanied by issue link(s) in comments in the surrounding code
- [ ] Tests are comprehensible, e.g. Ginkgo DSL is being used appropriately
- [ ] Docs updated or added to `/doc`
- [ ] Commit messages sensible and descriptive
- [ ] Tests marked as `[FLAKE]` are truly flaky and have an issue
- [ ] Code is properly formatted

### CODE REVIEW

File: test/e2e/installplan_e2e_test.go [hunk 1]
```diff
@@ -107,26 +108,74 @@ var _ = Describe("Install Plan", func() {
...
 			}
 
 			scheme := runtime.NewScheme()
-			Expect(policyv1beta1.AddToScheme(scheme)).To(Succeed())
+			Expect(deprecatedv1.AddToScheme(scheme)).To(Succeed())
 			{
 				var b bytes.Buffer
-				Expect(k8sjson.NewSerializer(k8sjson.DefaultMetaFactory, scheme, scheme, false).Encode(&pdb, &b)).To(Succeed())
+				Expect(k8sjson.NewSerializer(k8sjson.DefaultMetaFactory, scheme, scheme, false).Encode(&deprecated, &b)).To(Succeed())
 				manifest = b.String()
```
Comment (awgreene) [inline test/e2e/installplan_e2e_test.go:179]: If we can get rid of this scheme thing, we shouldn't need to import github.com/awgreene/deprecated-crd

File: test/e2e/installplan_e2e_test.go [hunk 2]
```diff
@@ -107,26 +108,74 @@ var _ = Describe("Install Plan", func() {
...
+									},
+								},
+							},
+						},
+					},
+				},
+				Status: apiextensionsv1.CustomResourceDefinitionStatus{
+					StoredVersions: []string{"v1"},
+				},
+			}
```
Comment (awgreene) [inline test/e2e/installplan_e2e_test.go:156]: Could move this to a test file for readability's sake.

File: test/e2e/installplan_e2e_test.go (hunk 1 above)
Comment (awgreene) [reply test/e2e/installplan_e2e_test.go:131]: Done!

File: test/e2e/installplan_e2e_test.go [hunk 3]
```diff
@@ -107,26 +102,21 @@ var _ = Describe("Install Plan", func() {
 					counter = metric.Value
 				}
 			}
+			deprecatedCRD, err := util.DecodeFile("./testdata/deprecated-crd/deprecated.crd.yaml", &apiextensionsv1.CustomResourceDefinition{})
```
Comment (timflannagan) [inline test/e2e/installplan_e2e_test.go:105]: There's a CLI flag for determining the testdata base directory: `testdataPath`.

Comment (awgreene) [reply test/e2e/installplan_e2e_test.go]: Thanks!

**Pull request merged**